src/transliteration/ui/LineLemmasContext.ts

```diff
@@ -1,6 +1,6 @@
 import React, { useContext } from 'react'
 import DictionaryWord from 'dictionary/domain/Word'
-import { LineToken } from './line-tokens'
+import { OneOfLineToken } from './line-tokens'
```

Comment:
Better use full paths, i.e. `'transliteration/ui/LineLemmasContext/line-tokens'`.

Reply:
That was an auto-import for sure. If we prefer absolute imports, maybe we should update the config so auto-imports are absolute by default.